scripts/travis_install

```diff
@@ -48,9 +48,10 @@ conda install --yes nodejs
 echo "node version $(node -v)"
 echo "npm version $(npm -v)"
 
+conda install --yes boto
```

Comment:
The report (run on py3.4 integration) successfully uploaded to s3: https://s3.amazonaws.com/bokeh-travis/104764781/report.html